app/views/categories/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>
+  <strong>Category:</strong>
+  <%= @category.title %>
+</h3>
+
+<h4>Create a <%= link_to 'new Post', new_category_post_path(@category) %></h4>
+
+<% @posts.each do |post| %>
+  <p><%= post.username %> posted <%= post.title %></p>
+<% end %>
+
```

Comment:
I want to make the title a link to a 'posts show page'. (Mayyyybe a User page with all posts from that User.)